webapp/src/components/App/Singlestudy/NavHeader/index.tsx

```diff
@@ -27,7 +27,7 @@ interface Props {
 
 function NavHeader({ study, parentStudy, variantTree, isExplorer }: Props) {
   const tree = variantTree && findNodeInTree(study.id, variantTree);
-  const variantNb = tree ? countDescendants(tree) : 0;
+  const variantNb = tree && countDescendants(tree);
```

Comment:
any particular reason to modify this ? With the previous version (`const variantNb = tree ? countDescendants(tree) : 0;`) it ensures `variantNb` is always a `number` (no `?` needed), and you can remove all the defensive checks.

Reply:
Because if `tree` we cannot have the number of variants, so 0 may be a lie

Reply:
Ok I see why you changed it, but it looks weird to me that tree can be undefined, are you sure ? `getVariantTree` in `variant.ts` returns `Promise<VariantTree>`, not `Promise<VariantTree | undefined>`.

Reply:
Shouldn't we throw an error instead in `findNodeInTree` when we request a study which is not in the tree ?
This would remove the `undefined` case

Reply:
I can keep the previous code, @hdinia was true.

@sylvlecl Normally it's not possible to have `undefined` here, we using the ternary instead of forcing the type with TypeScript.